CodingChallenge/Models/SearchDataSource.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  SearchDataSource.swift
+//  CodingChallenge
+//
+//  Created by Edgar Barocio on 6/23/19.
+//  Copyright © 2019 Edgar Barocio. All rights reserved.
+//
+
+import UIKit
+
+class SearchDataSource: NSObject, UICollectionViewDataSource {
+    
+    var results: [SongResponse] = []
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return results.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let identifier = "SearchResultCell"
```

Comment:
Add the reuseIdentifier to SearchResultsCell as a static var. Then you can use it here AND in the view controller where you're registering the reuseIdentifier.